vm/devices/get/guest_emulation_transport/src/error.rs

```diff
@@ -67,13 +67,13 @@ pub struct SaveRestoreOperationFailure {}
 /// Error while invoking an IgvmAttestRequest
 #[allow(missing_docs)] // self-explanatory fields
 #[derive(Debug, Error)]
-pub enum IgvmAttestError {
-    #[error("`agent_data` size {input_size} was larger than expected {expected_size}")]
+pub enum GuestEmulationTransportIgvmAttestError {
```

Comment:
One thing we certainly don't want to do is prefix error type names with the full name / path of the crate or module they are in.

Even if we do decide to include the type name in the error text, we should just use they type's fully-qualifed-path (e.g: in this case, `guest_emulation_transport::error::IgvmAttestError::Foo`, while leaving the type name the same as it was.

Reply:
Will address this in a separate PR